packages/static-config/src/swc.ts

```diff
@@ -0,0 +1,188 @@
+import {
```

Comment:
```suggestion
import type {
```

Are these all types?